src/state/api.js

```diff
@@ -14,11 +14,7 @@ const DEFAULT_SEARCH_RESULTS_LIMIT = 100; // maximum number of results to show
 
 export async function getRandomProbes(numProbes, process) {
   const data = await fetch(randomProbeURL, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify({ n: numProbes, process }),
+    method: 'GET',
```

Comment:
GET is the right method for this. Besides, django caching only works with HEAD or GET.